Lesson9/merve/styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+    box-sizing: border-box;
+  }
+  body{
+  max-width: 100%;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  font-size: 1em;
+  font-family:  medium-content-sans-serif-font, -apple-system, BlinkMacSystemFont,    "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-   serif;
+  font-weight: 400;
+  letter-spacing: 0.01em;
+  line-height: 1.4em;
+  }
+  h1{
+  font-family: sohne, "Helvetica Neue", Helvetica, Arial, sans-serif;
+   font-weight: 800;
+   font-size: 42px;
+   line-height: 52px;
+   letter-spacing: -0.011em;
+   color: #242424;
+   margin-top: 1.19em;
+   margin-bottom: 32px;
+  }
+  .first-box{
+   max-width: 700px;
+  }
+  .first-img{
+  border-radius: 50px;
+  position: absolute;
+  }
+  .second-img{
+  position: absolute;
+  margin-left: 30px;
+  margin-top: 15px;
+  border-radius: 50px;
+  border: 3px solid rgb(255, 255, 255);
+  }
+  a{
+  text-decoration: underline!important;
+  color:inherit;
+  font-weight: 500;
+  }
+  .profile, .next-info{
+  margin-left: 75px;
+  font-size: 14px;;
+  font-weight: 400;
+  line-height: 20px;
+  color: #242424;
+  }
+  .link {
+  color: inherit;
+  text-decoration: none;
+  }
+  #chart{
+  color:inherit;
+  }
+  .link:hover {
+  text-decoration: underline;
+  cursor: pointer;
+  }
+  .light {
+  color: #6B6B6B;
+  }
+  .btn {
+  color: rgb(66, 154, 53);
+  text-decoration: none;
+  cursor: pointer;
+  }
+  .divider {
+  border-bottom: 1px solid #6B6B6B;
+  padding-top: 30px;
+  }
+  .social-media-box{
+  height: 10px;
+  }
+  .clock{
+  display:flex;
+  align-items: center;
+  margin-top: 40px;
+  }
+  .images{
+  max-width:100%;
+  display: flex;
+  align-items:center;
+  justify-content: center;
+  }
+  figcaption{
+  display: flex;
+  align-items:center;
+  justify-content: center;
+  max-width: 728px;
+  margin-top: 10px;
+  margin-right: auto;
+  margin-left: auto;
+  text-align: center;
+  color: #6B6B6B;
+  font-size: 14px;
+  font-weight: 500;
+  }
+  .article, footer{
+  max-width: 60%;
+  margin: 50px auto;
+  letter-spacing: -0.003em;
+  line-height: 32px;
+  font-size: 20px;
+  font-family: source-serif-pro, Georgia, Cambria, "Times New Roman", Times, serif;
+  }
+  pre{
+  min-width: fit-content;
+  margin-bottom: -0.09em;
+  margin-top: -0.09em;
+  line-height: 1.5;
+  font-weight: 500;
+  font-family: source-code-pro, Menlo, Monaco, "Courier New", Courier, monospace;
+  letter-spacing: -0.022em;
+  font-size: 16px;
+  color: #000000;
+  background-color: #f2f2f2;
+  padding: 15px;
+  }
+  cite, .cite{
+  background-color: #f2f2f2;
+  padding: 5px;
+  font-style: normal;
+  line-height: 1.5;
+  font-weight: 500;
+  font-family: source-code-pro, Menlo, Monaco, "Courier New", Courier, monospace;
+  letter-spacing: -0.022em;
+  font-size: 16px;
+  color: #000000;
+  }
+  p{
+  margin: 40px 90px 40px 90px;
```

Comment:
Maybe you need vertical margin but certainly not the horizontal margin (90px). So I'd change it to `margin: 40px auto`